package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "test": "npm run spell-check",
+    "spell-check": "cspell \"**/*.html\" \"**/*.md\" \"**/*.js\" \"**/*.css\""
```

Comment:
Thoughts on pushing the file patterns into the cspell config? That ensures that running it directly on the command line or via an editor integration picks up only the "right" files. Unless it's intentional that IDE integrations check everything?
(I suspect I missed this on the website repo)

Reply:
👍 [9ad5664](https://github.com/srobo/docs/pull/320/commits/9ad566462e812aaefbdeac1a2ac511ce36f6d8e0)